### PR TITLE
Ct 186 Rename 'Terminate'

### PR DESCRIPTION
**Pull request by yeunga**

### CODE REVIEW

File: software-containers/arcade-casa/version-3.4-4.2/startup.sh [hunk 1]
```diff
@@ -1,8 +1,23 @@
 #!/bin/bash
 
-echo "applying leap second patch"
-/arcade/update-data.patch /opt/casa
```
Comment (brianmajor) [inline software-containers/arcade-casa/version-3.4-4.2/startup.sh:4]: I'm probably not up to date on this but just wondering why update-data.patch has been removed...

File: session-containers/arcade-desktop/software-scripts/nifty.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```
Comment (brianmajor) [inline session-containers/arcade-desktop/software-scripts/nifty.sh]: These scripts are created automatically on make, so please delete this file.

File: session-containers/arcade-desktop/software-scripts/Gemini-DRAGONS.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```
Comment (brianmajor) [inline session-containers/arcade-desktop/software-scripts/Gemini-DRAGONS.sh]: These scripts are created automatically on make, so please delete this file.

File: session-containers/arcade-desktop/arcade-software.properties [hunk 4]
```diff
@@ -37,3 +37,4 @@ casa-5.6.1-8 = canfar-registry.cloud.computecanada.ca/arcade-casa:5.6.1-8.el6
 #astrodebian = sfabbro/astrodebian:0.2.2
 python = canfar-registry.cloud.computecanada.ca/arcade-python:latest
 nifty = nat1405/nifty:0.1
+niri = nat1405/niri:0.1
```
Comment (brianmajor) [inline session-containers/arcade-desktop/arcade-software.properties]: Isn't one of these supposed to be named Gemini-DRAGONS now?

**Review by brianmajor — CHANGES_REQUESTED** [review]
The make process creates all the scripts that go into /usr/local/bin, so don't create custom ones.

The title of the xterm is being handled by the arcade service, so if there are any curl commands that post a param that is intented to be the title, please remove them.  I don't think there are any special differences between any of the launching scripts so their creation on make is the same.

File: software-containers/arcade-casa/version-3.4-4.2/startup.sh (hunk 1 above)
Comment (yeunga) [reply software-containers/arcade-casa/version-3.4-4.2/startup.sh:4]: The update-data.patch was suggested by CASA in its out-of-date document to fix the leap second issue. We are using /cavern/projects/casa-data-repository. The patch file is not needed anymore.

File: session-containers/arcade-desktop/software-scripts/nifty.sh (hunk 2 above)
Comment (yeunga) [reply session-containers/arcade-desktop/software-scripts/nifty.sh]: The generated nifty.sh uses param and that was the reason for the start up failure. I suppose I can add another template and add some conditional code in create-software-links.sh to generate nifty.sh correctly. Just let me know if you prefer adding another template.

File: session-containers/arcade-desktop/arcade-software.properties (hunk 4 above)
Comment (yeunga) [reply session-containers/arcade-desktop/arcade-software.properties]: Updated niri to Gemini-DRAGONS.

**Comment by yeunga** [comment]
Currently create-software-links.sh uses software-sh.template to generall the .sh files. Hence all generated .sh files use param except for the custom .sh files. xterm, nifty and Gemini-DRAGONS use their own custom .sh files.

File: session-containers/arcade-desktop/software-scripts/Gemini-DRAGONS.sh (hunk 3 above)
Comment (yeunga) [reply session-containers/arcade-desktop/software-scripts/Gemini-DRAGONS.sh]: ditto

**Review by brianmajor — APPROVED** [review]

**Pull request merged**